### PR TITLE
hide actions

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: actions/SasImport.php [hunk 1]
```diff
@@ -20,6 +20,8 @@
  * 
  */
 namespace oat\taoTestTaker\actions;
+use oat\tao\model\routing\AnnotationReader\security;
```
Comment (martijn-tao) [inline actions/SasImport.php]: Please leave one newline between the namespace, and imports

File: actions/TestTaker.php [hunk 2]
```diff
@@ -44,11 +45,16 @@
 class TestTaker extends tao_actions_SaSModule
 {
     use OntologyAwareTrait;
-    use EventManagerAwareTrait;
+
+    protected function getEventManager()
```
Comment (martijn-tao) [inline actions/TestTaker.php]: Please add a docblock

**Comment by zagovorichev** [comment]
@julien-sebire Yes, maybe. But I don't think that I need to create new branches: 1. there are a lot of PRs which were already merged and this change will be misleading. 2. This branch will be deleted after the merge, so how much sense in this renaming?

**Comment by julien-sebire** [comment]
I agree with you, I'm quite new to the process and I guess it will just have to be amended in the release of the extensions... Well, no big a deal anyway! ;-)

**Pull request merged**